harmonica/_forward/tesseroid.py

```diff
@@ -201,9 +203,11 @@ def tesseroid_gravity(
             dtype,
             progress_proxy,
         )
-    result *= GRAVITATIONAL_CONST
+    # Invert sign
+    if field in ("g_z"):
+        result *= -1
     # Convert to more convenient units
-    if field == "g_z":
+    if field in ("g_z"):
```

Comment:
Just a minor suggestion: if we don't add the comma, then the _collection_ for the `in` statement is just the `"g_z"` string, and not a tuple containing only the `"g_z"` string.
```suggestion
    if field in ("g_z", ):
```